packages/crud/test/crud-buttons.test.js

```diff
@@ -17,17 +17,46 @@ describe('crud buttons', () => {
         if (mode === 'default') {
           crud = fixtureSync('<vaadin-crud style="width: 300px;"></vaadin-crud>');
         } else {
-          crud = fixtureSync(
-            `<vaadin-crud style="width: 300px;">
+          crud = fixtureSync(`
+            <vaadin-crud style="width: 300px;">
               <vaadin-button slot="save-button"></vaadin-button>
               <vaadin-button slot="cancel-button"></vaadin-button>
               <vaadin-button slot="delete-button"></vaadin-button>
-            </vaadin-crud>`
-          );
+            </vaadin-crud>
+          `);
         }
         crud.items = [{ foo: 'bar' }];
         await nextRender(crud._grid);
-        [btnSave, btnCancel, btnDelete] = crud.querySelectorAll('vaadin-button');
+        [saveButton, cancelButton, deleteButton] = crud.querySelectorAll('vaadin-button');
+      });
+
+      describe('i18n', () => {
+        it('should set the label for the delete button', async () => {
```

Comment:
**note:** Adding these tests indicated that the i18n button labels didn't use to be correctly set during initialization before. The labels used to be in place just by coincidence, because of the `__isNew`, `__isDirty` property change.